docs/csharp/nullable-references.md

```diff
@@ -42,7 +42,7 @@ Any reference type can have one of four *nullabilities*, which describes when wa
 
 - *Nonnullable*: Null can't be assigned to variables of this type. Variables of this type don't need to be null-checked before dereferencing.
 - *Nullable*: Null can be assigned to variables of this type. Dereferencing variables of this type without first checking for `null` causes a warning.
-- *Oblivious*: This is the pre-C# 8.0 state. Variables of this type can be dereferenced or assigned without warnings.
+- *Oblivious*: This is the pre-C# 8.0 state. Variables of this type can be dereferenced or assigned to without warnings.
```

Comment:
On second thought, I think the original text is more correct. A variable is assigned a value. A value is assigned to a variable. So I think that "variables...can be...assigned without warnings" is valid. Thoughts?

Reply:
After reading the original text again, I think it depends on how it's interpreted.

Original text is:

> Variables of this type can be dereferenced or assigned without warnings.

The two interpretations with different meanings are:

> (Variables of this type can be (dereferenced or assigned) without warnings.

> (Variables of this type can ((be dereferenced) or (assigned)) without warnings.

I'm not sure if I can express what I mean well by that.

Reply:
Had to read it several times 😆 .
Yes, it seems the original text is correct.

It says: "Variables can be assigned", which seems correct to me. I'm going to revert this change.

```suggestion
- *Oblivious*: This is the pre-C# 8.0 state. Variables of this type can be dereferenced or assigned without warnings.
```